Source/Core/Core/IOS/USB/Emulated/Skylander.cpp

```diff
@@ -19,465 +19,555 @@
+     {"Krypt King (Nitro) (Trap Master)", Game::TrapTeam, Element::Undead, Type::Skylander}},
+    {{479, 0x3000}, {"Short Cut (Trap Master)", Game::TrapTeam, Element::Undead, Type::Skylander}},
+    {{480, 0x3000}, {"Bat Spin", Game::TrapTeam, Element::Undead, Type::Skylander}},
+    {{481, 0x3000}, {"Funny Bone", Game::TrapTeam, Element::Undead, Type::Skylander}},
+    {{482, 0x3000},
+     {"Knight Light (Trap Master)", Game::TrapTeam, Element::Light, Type::Skylander}},
+    {{483, 0x3000}, {"Spot Light", Game::TrapTeam, Element::Light, Type::Skylander}},
+    {{484, 0x3000}, {"Knight Mare (Trap Master)", Game::TrapTeam, Element::Dark, Type::Skylander}},
+    {{485, 0x3000}, {"Blackout", Game::TrapTeam, Element::Dark, Type::Skylander}},
+    {{502, 0x3000}, {"Bop (Mini)", Game::TrapTeam, Element::Earth, Type::Skylander}},
+    {{503, 0x3000}, {"Spry (Mini)", Game::TrapTeam, Element::Magic, Type::Skylander}},
+    {{504, 0x3000}, {"Hijinx (Mini)", Game::TrapTeam, Element::Undead, Type::Skylander}},
+    {{505, 0x3000}, {"Terrabite (Mini)", Game::TrapTeam, Element::Earth, Type::Skylander}},
+    {{506, 0x3000}, {"Breeze (Mini)", Game::TrapTeam, Element::Air, Type::Skylander}},
+    {{507, 0x3000}, {"Weeruptor (Mini)", Game::TrapTeam, Element::Fire, Type::Skylander}},
+    {{507, 0x3402},
+     {"Weeruptor (Eggsellent) (Mini)", Game::TrapTeam, Element::Fire, Type::Skylander}},
+    {{508, 0x3000}, {"Pet Vac (Mini)", Game::TrapTeam, Element::Air, Type::Skylander}},
+    {{508, 0x3402},
+     {"Pet Vac (Power Punch) (Mini)", Game::TrapTeam, Element::Air, Type::Skylander}},
+    {{509, 0x3000}, {"Small Fry (Mini)", Game::TrapTeam, Element::Fire, Type::Skylander}},
+    {{510, 0x3000}, {"Drobit (Mini)", Game::TrapTeam, Element::Tech, Type::Skylander}},
+    {{514, 0x3000}, {"Gill Runt (Mini)", Game::TrapTeam, Element::Water, Type::Skylander}},
+    {{519, 0x3000}, {"Trigger Snappy (Mini)", Game::TrapTeam, Element::Tech, Type::Skylander}},
+    {{526, 0x3000}, {"Whisper Elf (Mini)", Game::TrapTeam, Element::Life, Type::Skylander}},
+    {{540, 0x1000}, {"Barkley (Sidekick)", Game::Giants, Element::Life, Type::Skylander}},
+    {{540, 0x3000}, {"Barkley (Mini)", Game::TrapTeam, Element::Life, Type::Skylander}},
+    {{540, 0x3402}, {"Barkley (Gnarly) (Mini)", Game::TrapTeam, Element::Life, Type::Skylander}},
+    {{541, 0x1000}, {"Thumpling (Sidekick)", Game::Giants, Element::Water, Type::Skylander}},
+    {{541, 0x3000}, {"Thumpling (Mini)", Game::TrapTeam, Element::Water, Type::Skylander}},
+    {{542, 0x1000}, {"Mini Jini (Sidekick)", Game::Giants, Element::Magic, Type::Skylander}},
+    {{542, 0x3000}, {"Mini Jini (Mini)", Game::TrapTeam, Element::Magic, Type::Skylander}},
+    {{543, 0x1000}, {"Eye-Small (Sidekick)", Game::Giants, Element::Undead, Type::Skylander}},
+    {{543, 0x3000}, {"Eye-Small (Mini)", Game::TrapTeam, Element::Undead, Type::Skylander}},
+    {{999, 0x1000}, {"Debug Minion", Game::SpyrosAdv, Element::Magic, Type::Skylander}},
```

Comment:
Didn't think Debug Minion was in the Wii versions of the game?

Reply:
That one shouldn’t be in there either way. Feel like this list should only contain official skylanders.